src/main/java/io/vertx/openapi/contract/OpenAPIContract.java

```diff
@@ -195,11 +196,21 @@ static Future<OpenAPIContract> from(Vertx vertx, JsonObject unresolvedContract,
    * @param method  The method of the request.
    * @return the found {@link Operation} object, or null if the passed path and method doesn't match any {@link Operation} object.
    */
+  @Nullable
   Operation findOperation(String urlPath, HttpMethod method);
 
   /**
    * Returns the applicable list of global security requirements (scopes) or empty list.
    * @return The related security requirement.
    */
   List<SecurityRequirement> getSecurityRequirements();
+
+  /**
+   * Finds the related {@link SecurityScheme} object based on the passed name.
+   *
+   * @param name The name of the security scheme.
+   * @return the found {@link SecurityScheme} object, or null if the passed path and method doesn't match any {@link Operation} object.
+   */
+  @Nullable
+  SecurityScheme findSecurityScheme(String name);
```

Comment:
`securityScheme(String name)` to be similar to operation(String operationId).

Should we also add a `List<SecurityScheme> getSecuritySchemes()` method?